src/models/stdlib/Stdlib.js

```diff
@@ -22,6 +23,11 @@ export default class Stdlib {
     this._packageJson = fs.parseJson(filename)
     return this._packageJson
   }
+
+  hasContract(alias) {
+    if (!this.getPackage().contracts) return false;
+    return !_.isEmpty(this.getPackage().contracts[alias]);
```

Comment:
We should modify the Object prototype and stop using lodash 🙏

Reply:
We are going to have a heated discussion I'm afraid. I love lodash, and have a deeply rooted hate for modifying prototypes (probably because of several years of monkeypatching metaclasses in Ruby).

Reply:
How about we switch to Clojure, which includes all lodash-like methods in its stdlib?